config.py

```diff
@@ -90,7 +90,9 @@ def init_app(app):
         app.config['INDEXD_USER'] = indexd_secrets['data']['user']
         app.config['INDEXD_PASS'] = indexd_secrets['data']['password']
 
-        app.config['BUCKET_SERVICE_TOKEN'] = bucket_secrets['data']['token']
+        if bucket_secrets:
+            app.config['BUCKET_SERVICE_TOKEN'] = \
+                    bucket_secrets['data']['token']
```

Comment:
Will data and token keys always exist?

Reply:
They should as long as bucket secrets comes back non-null

Reply:
But I will make it more defensive